### PR TITLE
add easy install instructions

### PR DESCRIPTION
**Pull request by jonathanspw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ For format patterns, libraries, magic and constant files, check out the [ImHex-P
 
 To use ImHex, the following minimal system requirements need to be met:
 
-- **OS**: Windows 7 or higher, macOS 10.10 (Yosemite) or higher, "Modern" Linux (Ubuntu 22.04+, Fedora Stable/Rawhide and Arch Linux are officially supported)
+- **OS**: Windows 7 or higher, macOS 10.10 (Yosemite) or higher, "Modern" Linux (Ubuntu 22.04+, Fedora Stable/Rawhide, and Arch Linux are officially supported)
```
Comment (iTrooz) [inline README.md:119]: I'd propose to phrase this to say these distributions have a *dedicated package*, as we still provide support for other distributions through the AppImage

Comment (jonathanspw) [reply README.md:119]: All I did was add a comma.  I didn't otherwise change this text that someone else wrote.

**Comment by WerWolv** [comment]
Thank you. I feel like we should maybe add a new file for this (maybe a `INSTALL` file in the root of the repo or a file in the `/dist` folder) since the list of distros and install methods is already pretty big and will most likely grow in the future.

**Comment by iTrooz** [comment]
I propose to merge this for now, and possibly make add my suggestions in another PR, since they weren't related to the changes

**Pull request merged**